### ISSUE TITLE
Should Multiple Concepts be Allowed?

### ISSUE THREAD

**Issue by cmoesel** [standardhealth/shr_spec#9]
From @gregquinn2001: Allowing multiple concepts for an entry seems counter to SHR goals to me. Do we really need to allow multiple concepts?

**Comment by cmoesel** [comment]
From @markkramerus: Probably not, but in some cases, it is just a matter of the shortcomings in MTH. For example, this morning I was looking for frequency terms, and found this:

``` 
C1549495  Never (AcknowledgementCondition)
C2003901  Never (frequency)
C4036128  No, never
C3843461  Never or nearly never
C3844158  Almost never or never
```

To me, these are hairs not worth splitting. Should we pick one arbitrarily, or list the synonyms? I’m OK with picking one for now.

**Comment by jsnichols** [comment]
This is the advantage of using MTH concepts; if you include the MTH CUI, then you link to related concepts across all MTH coding systems.

I feel strongly we should use MTH CUIs whenever possible. For situations where this is not possible, we should set some guard rails.

**Comment by cmoesel** [comment]
This is also discussed in #12.

**Comment by markkramerus** [comment]
Closed, see #12 for explanation.
